website/src/pages/batch-changes.tsx

```diff
@@ -0,0 +1,244 @@
+import { PageProps } from 'gatsby'
+import * as React from 'react'
+import { ContentSection } from '../components/content/ContentSection'
+import Layout from '../components/Layout'
+import Tab from 'react-bootstrap/Tab'
+import Tabs from 'react-bootstrap/Tabs'
+import { TrySourcegraph } from '../components/TrySourcegraph'
+import { RequestDemoAction } from '../css/components/actions/RequestDemoAction'
+
+const Video: React.FunctionComponent<{ name: string }> = ({ name }) => (
+    <video
+        className="w-100 h-auto shadow"
+        width={1280}
+        height={720}
+        autoPlay={true}
+        muted={true}
+        loop={true}
+        playsInline={true}
+        controls={false}
+    >
+        <source
+            type="video/webm"
+            src={`https://storage.googleapis.com/sourcegraph-assets/batch-changes/${name}.webm`}
+        />
+        <source type="video/mp4" src={`https://storage.googleapis.com/sourcegraph-assets/batch-changes/${name}.mp4`} />
+    </video>
+)
+
+export const BatchChangesPage: React.FunctionComponent<PageProps> = props => (
+    <Layout
+        location={props.location}
+        meta={{
+            title: 'Sourcegraph - Batch Changes',
+            description: 'Automate large-scale code changes',
+            image: '/favicon.png',
+        }}
+        heroAndHeaderClassName="batch-changes-page__hero-and-header"
+        hero={
+            <div className="container pb-4">
+                <div className="row">
+                    <div className="col-lg-7 mb-lg-6 mt-6">
+                        <div className="text-uppercase">Batch Changes</div>
+                        <h1 className="display-2 font-weight-bold mb-0">Automate large-scale code&nbsp;changes</h1>
+                        <p className="home__semiwide-paragraph my-5 display-4">
+                            Keep your code up to date, fix critical security issues, and pay down tech debt across all
+                            of your repositories with Batch Changes.
+                        </p>
+                        <RequestDemoAction />
```

Comment:
@attfarhan @felixfbecker - we have decided to remove the email form on the Batch Changes landing page. We just want a CTA button that says "Request a demo" and links to the page @attfarhan created. Thanks! CC @lorihey @sarah-reece